ctp2_code/gs/fileio/civscenarios.cpp

```diff
@@ -461,11 +461,14 @@ CIV_SCEN_ERR CivScenarios::MakeNewPack(MBCHAR *dirName, MBCHAR *packName, MBCHAR
 	g_civPaths->GetScenarioRootPath(rootPath);
 
 	sprintf(path, "%s%s%s", rootPath, FILE_SEP, dirName);
+
+	int rc;
 #ifdef WIN32
-	if(!_stat(path, &tmpstat)) {
+	rc = _stat(path, &tmpstat);
 #else
-	if(!stat(path, &tmpstat)) {
+	rc = stat(path, &tmpstat);
 #endif
+	if (rc == 0) {
```

Comment:
Same here, does the new logic also fit the return value of `_stat(` from windows?

Reply:
Yes.